gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-2-menu-item/step-2-menu-item.component.scss

```diff
@@ -0,0 +1,23 @@
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+.step-2-menu-item {
+  display: flex;
+  flex-direction: column;
+  font-weight: bold;
+  padding: 16px;
+  position: relative;
```

Comment:
I wonder if it is necessary to have Pos: relative and Pos absolute ? maybe we can juste have a padding for line no ?

Reply:
I don't think, but we can try

Reply:
as you wish I was just wondering. if not 🚀